test_data/encodingTests.csv

```diff
@@ -23,3 +23,5 @@ CFX30000+,92,1,89,1,90,2
 62H20000+,1,180,1,-180,2,-179
 62H30000+,1,181,1,-179,2,-178
 CFX3X2X2+X2,90,1,89.9998750,1,90,1.0001250
+6GFRP39C+5HG4QWR,-0.2820710399999935,36.07145996093760,-0.2820710399999935,36.07145996093752,-0.2820709999999935,36.07146008300783
```

Comment:
Thanks for adding some more test cases! Unfortunately, all the other implementations are failing since they don't have these changes yet. Can you revert these changes for now? I'll add some test cases when the other implementations can pass these cases.

Reply:
Ok